tests/Tests/MenuItemTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace JK\Mittagstisch\Tests;
+
+use JK\Mittagstisch\MenuItem;
+
+class MenuItemTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var MenuItem */
+    private $instance;
+
+    public function setUp()
+    {
+        $this->instance = new MenuItem('Menüname', 7.77);
+    }
+
+    public function testInstance()
+    {
+        static::assertInstanceOf('JK\Mittagstisch\MenuItem', $this->instance);
```

Comment:
> * A regular user of PHPUnit should use $this-> instead of self::

Source: https://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-148407321